### DISCUSSION TITLE
Optimized reconstruction of  crystallographic ODF

### DISCUSSION THREAD

**Discussion by ghost** [mtex-toolbox/mtex#1545] [Ask Anything]
Hello,

I'm making crystal plasticity simulations, and for that purpose I would like to sample real crystallographic orientations data as accurately as possible, for a given number of orientations. This is exactly the topic of the paper "Optimized reconstruction of the crystallographic orientation density function based on a reduced set of orientations",  [here](https://doi.org/10.1107/S1600576719017138)

In MTEX, as far as I understand, odf.discreteSample() is random. In the above paper, it is said that the optimization algorithm is implemented in MTEX, but I cannot find it in the documentation. Has it been released ? I'm not confident enough in texture to implement it myself, so if the optimization has been implemented in METX, it would be a life saver.

Thank you.

Aurélien

**Comment by ralfHielscher** [comment]
Hi Aurélien,

I contacted the coauthors and asked them whether thy can share the code. It is not officially included into MTEX.

Ralf.

**Reply by ghost** [reply]
ok, thank you for the info and for asking !
